tests/layer_tests/pytorch_tests/test_pad.py

```diff
@@ -244,3 +244,121 @@ def test_reflection_padnd(self, pads, dtype, ie_device, precision, ir_version):
         print(ndim)
         self._test(*self.create_model(pads), ie_device, precision, ir_version,
                    kwargs_to_prepare_input={"ndim": ndim, "dtype": dtype})
+
+class TestReplicatePad1D(PytorchLayerTest):
+    def _prepare_input(self, ndim=4, dtype="float32"):
+        import numpy as np
+        input_5d_shape = [5,9,1,1,2,4]
```

Comment:
please remove `input_5d_shape` and `ndim` from here and add normal `input_shape` to `@pytest.mark.parametrize`.
The same comment for other tests

Reply:
done,
please suggest the input_shapes to keep currently kept random from my end,
thank you